en/docs/learn/examples/data-integration/mongo-data-service.md

```diff
@@ -1,4 +1,4 @@
-# Exposing a Mongo Datasource
+# How to expose a Mongo Datasource
```

Comment:
Let's use title-case capitalization for the main title. Please check the other pages as well.